scprep/run/dyngen.py

```diff
@@ -0,0 +1,241 @@
+import pandas as pd
+from . import r_function
+
+_install = r_function.RFunction(
+    args="""lib=.libPaths()[1], dependencies=NA, verbose=TRUE""",
+    body="""
+        cat ('Installing package in:', lib, '\n')
+        quiet <- !verbose
+
+        if (!require('remotes')) install.packages('remotes')
+        remotes::install_github('dynverse/dyngen',
+                         repos = "http://cran.us.r-project.org",
+                         lib=lib, dependencies=dependencies,
+                         quiet=quiet)
+    """,
+)
+
+_get_backbones = r_function.RFunction(
+    args="""lib=.libPaths()[1]""",
+    body="""
+        library(dyngen, lib.loc=lib)
```

Comment:
Use `setup`, as in https://github.com/KrishnaswamyLab/scprep/blob/master/scprep/run/splatter.py#L45. Also, is it necessary to specify `lib`? I lean towards adding to `.libPaths()` over using a custom `lib.loc` for a specific package.